extensions/azurecore/src/account-provider/azureAccountProviderService.ts

```diff
@@ -147,8 +147,8 @@ export class AzureAccountProviderService implements vscode.Disposable {
 		const platform = os.platform();
 		const tokenCacheKey = `azureTokenCache-${provider.metadata.id}`;
 		const lockOptions = {
```

Comment:
While you're here you should give this an actual type (or I'd personally just move it down directly into the PersistenceCachePlugin constructor directly since that's the only placed it's used)